src/ert/config/gen_kw_config.py

```diff
@@ -41,6 +62,108 @@ def __post_init__(self) -> None:
         for e in self.transfer_function_definitions:
             self.transfer_functions.append(self._parse_transfer_function(e))
 
+        self._validate()
+
+    @classmethod
+    def from_config_list(cls, gen_kw: List[str]) -> Self:
+        gen_kw_key = gen_kw[0]
```

Comment:
Why are some errors here and some in _validate?

Reply:
Errors in `from_config_list` should be errors related to making the string in config into the parameters of the `GenKw` constructor. The errors in `GenKw` constructor (`_validate`) checks whether you have a valid object, ie. it relates to the transaction:

```python
try:
    genkw = GenKwConfig(...)
catch Exception err:
    # here, genkw is not initialized
# here you have a valid genkw object.